darwin/exporter/formats/darwin_1_0.py

```diff
@@ -1,34 +1,71 @@
 from pathlib import Path
-from typing import Any, Dict, Iterable, Union
+from typing import Any, Dict, Iterable, List, Union
 
 import orjson as json
 
-import darwin.datatypes as dt
+from darwin.datatypes import (
+    Annotation,
+    AnnotationAuthor,
+    AnnotationClass,
+    AnnotationFile,
+    DictFreeForm,
+    SubAnnotation,
+    VideoAnnotation,
+)
+from darwin.exceptions import (
+    ExportException,
+    ExportException_CouldNotAssembleOutputPath,
+    ExportException_CouldNotBuildOutput,
+    ExportException_CouldNotWriteFile,
+)
+
+
+def export(annotation_files: Iterable[AnnotationFile], output_dir: Path) -> None:
+    errors: List[Exception] = []
 
-
-def export(annotation_files: Iterable[dt.AnnotationFile], output_dir: Path) -> None:
     for id, annotation_file in enumerate(annotation_files):
-        _export_file(annotation_file, id, output_dir)
+        try:
+            _export_file(annotation_file, id, output_dir)
+        except Exception as e:
+            errors.append(e)
+
+    if errors:
+        raise ExportException.from_multiple_exceptions(errors)
+
+
+def _export_file(annotation_file: AnnotationFile, _: int, output_dir: Path) -> None:
+
```

Comment:
As well as the basic fix, I also made the function catch and encapsulate errors that do occur, which improves user experience if things fail